config/initializers/01_constants.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Constants
+  SEED_ADMIN_EMAIL = ENV.fetch("SEED_ADMIN_EMAIL", "admin@example.com")
```

Comment:
I would prefer to use `fetch` without fallback and keep the defaults in `.env.development`.
That way the app will fail in production if they are missing.